ui/app/src/components/PublishDialog/PublishDialogContainer.tsx

```diff
@@ -83,68 +83,70 @@ export function PublishDialogContainer(props: PublishDialogContainerProps) {
       publishingTarget: null
     };
 
-    let itemsChecked = items.filter((item) => selectedItems[item.path]);
+    if (detailedItems) {
+      let itemsChecked = detailedItems.filter((item) => selectedItems[item.path]);
```

Comment:
Wouldn't iterating selectedItems be more efficient? ie. less iterations, same result?
```suggestion
      let itemsChecked = selectedItems.map((item) => detailedItems[item.path]);
```
What's the relation of existence of items in one or the other?

Reply:
Also a filtering of the selected items lookup would be needed, since the value may be true/false:
`Object.keys(selectedItems).filter((path) => selectedItems[path]).map((path) => detailedItems[path]);`